soap-python-client.py

```diff
@@ -23,3 +25,15 @@
             })
 
 print(result)
+
+# upload demo
+with open ("./LICENSE", "rb") as in_file:
+
+    result = client.service.file_upload({
+        'fileName': f"karolsticker{random.random()}.png",
+        'fileContent': in_file.read(),
+        'location': 'None',
+        'token': 'eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE2OTgwNTc1MTYsInV1aWQiOiIyNGVlOWU4Ni0zNmFjLTQyNWMtYWU4Mi04Nzk3MjFmODI5NTMifQ.0IlrR1KTNa_SnJwvDsf-q9TkmP_DrEaPCP-1xVVSsQArL3-AHjp0YJETaS4sQjSNtkzhN2Fz0WkEqV-TnDDXCQ'
```

Comment:
Does this is deterministic? For example running the client script in a fresh environment will successfully pass all its tests?

Reply:
[soap-python-client.py](https://github.com/hawks-atlanta/gateway-java/pull/42/files/d809f030b14f626212a33c96cc294199bee9074d#diff-183c06352659a8e335d564adda4092cd120cca9010b3b1d67f182fbeae1760a6) is not a test. It's a file to help developers debug their SOAP methods. Only for reference, not deterministic, those methods will mostly fail.

Do you think we should keep it?

Reply:
Then ignore this comment